CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19)
```

Comment:
Can we specify a lower version of CMake? Are we depending on features that arrived in 3.19?

Reply:
I'll lower the version requirement

Reply:
Selected 3.10 as we are not using any fancy new features. The choice is kind of random, I just picked the version found on Ubuntu 18.04 which is the oldest LTS release currently supported.